src/methods/common/selfDescription.2210vp.service.ts

```diff
@@ -38,6 +38,22 @@ export class SelfDescription2210vpService {
   ) {}
 
   public async validate(typedVerifiablePresentation: TypedVerifiablePresentation): Promise<validationResultWithoutContent> {
+    let isValidSignature = await this.proofService.validateVP(
```

Comment:
@nklomp here and in the following lines we're verifying every VC signature that we receive.